examples/flutter_gallery/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
   intl: 0.15.1
   string_scanner: 1.0.2
   url_launcher: 0.4.2+5
+  cupertino_icons: ^0.1.0
```

Comment:
make sure to run `flutter update-packages --force-upgrade`

Reply:
Done, it brought in a bunch of file diffs

Reply:
I think it's messing with various tests. I'm gonna extract this into a separate PR.

Reply:
Rebased to head where Yegor already did this upgrade